configure

```diff
@@ -550,6 +550,26 @@ else
 fi
 print_config "Static build" "$build_static"
 
+##########################################
+# check for C11 atomics support
+cat > $TMPC <<EOF
+#include <stdatomic.h>
+int main(void)
+{
+  _Atomic unsigned v;
+  atomic_load(&v);
+  return 0;
+}
+EOF
+if ! compile_prog "" "" "C11 atomics"; then
+  echo
+  echo "From fio 3.21 compiler support for C11 atomics is required."
```

Comment:
I'd remove the version info here, it isn't really relevant. It'll just end up being a historic thing that's then removed later, and it doesn't even really add anything right now. They obviously have this version (or newer) if they run into this :-)

Reply:
Fair point - done.